Source/ZenLib/Ztring.cpp

```diff
@@ -1312,14 +1312,15 @@ Ztring& Ztring::Date_From_Seconds_1970 (const int32s Value)
 Ztring& Ztring::Date_From_Seconds_1970 (const int64s Value)
 {
     time_t Time=(time_t)Value;
-    #if _POSIX_C_SOURCE >= 1 || _XOPEN_SOURCE || _BSD_SOURCE || _SVID_SOURCE || _POSIX_SOURCE
+    #if defined(HAVE_GMTIME_R)

```

Comment:
Please keep the other `#if` (no change for people using CMake and so on)

Reply:
The other `if` works on GLIBC by accident, it isn't correct. Someone on musl and using CMake will have the same issue we did (unless we add the function check to CMake, which is probably best).

Reply:
OK, maybe less problematic for everyone if we go directly to the step with CMake implementation.
@g-maxime if not long for you please hint about CMake changes to do (I found [the WebKit corresponding test](https://trac.webkit.org/changeset/206496/webkit), maybe really easy to add in CMake, and @ericonr note the even a modern project like WebKit checks the presence of such function ;-) )

Reply:
Ok, I have added the CMake and the autoconf checks! Let me know what you think.

Reply:
I hadn't seen your second response, and it appears WebKit has a function of their own for adding function definitions. I hope my implementation is enough.